tests/tracing/test_disordered_tracing.py

```diff
@@ -1470,16 +1470,40 @@ def test_prep_arrays() -> None:
     """Test of prep_arrays()."""
 
 
-@pytest.mark.skip(reason="Awaiting test to be written 2024-10-15.")
-def test_grain_anchor() -> None:
+@pytest.mark.parametrize(
+    ("array_shape", "bounding_box", "pad_width", "target"),
+    [
+        pytest.param((3, 3), [1, 1, 2, 2], 1, (0, 0), id="Simple square padded by 1 to edges"),
+        pytest.param((3, 3), [1, 1, 2, 2], 2, (0, 0), id="Simple square padded by 2 beyond both edges"),
+        pytest.param((3, 10), [1, 1, 2, 2], 4, (0, 0), id="Simple square padded by 4 beyond rows but within columns"),
+        pytest.param((10, 10), [7, 7, 9, 9], 4, (3, 3), id="Simple square padded by 4 beyond bottom and right edges"),
+    ],
+)
+def test_grain_anchor(array_shape: tuple, bounding_box: list, pad_width: int, target: npt.NDArray) -> None:
```

Comment:
I'm not sure, I recall I introduced it because we needed a way of putting tracing, done on cropped grains, back into the original image and the solution I came up with was to have an anchor point so we could put it back in the right location.

If its been removed in the refactor in favour of an alternative method of putting everything back that is perfectly fine but there was too much in the code/pull request for me to spot that.

Reply:
Well it's tested and imo potentially useful so let's keep it. Was just wondering 😄